boards/nrf52dk/Makefile.include

```diff
@@ -15,6 +15,12 @@ export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
 export JLINK_FLASH_ADDR := 0x1f000
 export LINKER_SCRIPT ?= $(RIOTCPU)/$(CPU)/ldscripts/$(CPU_MODEL)_sd.ld
 endif
+
+export ROM_SIZE = 0x80000
+export SLOT0_SIZE = 0x8000
+export SLOT1_SIZE = 0x3C000
+export SLOT2_SIZE = 0x3C000
```

Comment:
I think I got confused on which variables I should put on either board or cpu code, but now I think everything is board-independent, so I can move this values safely.